### PR TITLE
V1.1.0

### PR DESCRIPTION
**Pull request by jpgarbanzo**

### CODE REVIEW

File: src/blocks/base-ui.js [hunk 1]
```diff
@@ -332,7 +332,7 @@ define([
                     // isEmpty can be undefined
                     // ..but promise AND isEmpty can't be processed
                     if (typeof childEmptinessPromise === 'undefined' && childEmptinessUndefined) {
-                        Logger.error('base-ui : checkLocalEmptiness : unkown empty state or promise', childrenEmptiness);
+                        Logger.error('base-ui : checkLocalEmptiness : unkown empty state or promise', childResult);
```
Comment (andreybs11) [inline src/blocks/base-ui.js]: Why the rename?

File: src/data-flow/fetch/binder.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+        this.fetchHandler = new FetchHandler({
+            prepareDataForFetch : this.opts.prepareDataForFetch,
+            proccessResponse : this.opts.proccessResponse,
+            fetchCallback : this.opts.fetchCallback
+
+            // proxy : this.opts.proxy,
+            // // if this opt is provided, `FetchHandler` will attempt to call `proxy[fetchProxyCallbackName]`
+            // // otherwise, we'll assume `proxy.fetch` is the handler
+            // fetchProxyCallbackName : this.opts.fetchProxyCallbackName
```
Comment (andreybs11) [inline src/data-flow/fetch/binder.js]: fetchProxyCallbackName? legacy documentation?

File: src/data-flow/save/binder.js [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+         */
+        _onUiSaveRequest : function(successCallback, exitOnSuccessCallback, errorCallback) {
+            this.saveHandler.saveData(this.opts.component)
+                .then(
+                    $.proxy(this._performSaveSuccessActions, this, successCallback, exitOnSuccessCallback),
+                    $.proxy(this._performSaveErrorActions, this, errorCallback)
+                );
+        },
+
+        _performSaveSuccessActions : function(uiSuccessCallback, exitOnSuccessCallback, response) {
```
Comment (andreybs11) [inline src/data-flow/save/binder.js]: Documentation

File: src/blocks/base-ui.js (hunk 1 above)
Comment (jpgarbanzo) [inline src/blocks/base-ui.js]: dunno, it was like that in the branch I've used as base

**Comment by andreybs11** [comment]
Pancho Villa!
You owe me the rename

**Pull request merged**